src/Common/src/Internal/Runtime/TypeLoader/EETypeFlags.cs

```diff
@@ -101,4 +101,58 @@ internal enum EETypeKind : ushort
         /// </summary>
         GenericTypeDefEEType = 0x0003,
```

Comment:
I think that the original directory `Internal/Runtime` was more appropriate - for layering. Runtime.Base should not depend on TypeLoader, but it does with this change as far as the naming goes.